### PR TITLE
Add option "margin_references" to tufte_features.

### PR DESCRIPTION
**Pull request by stefanfritsch**
Hi,

I need a document with the references at the end of the document instead of in margin notes. So I added a tufte_feature for the citations.

I'll send the contributor agreement shortly even though I don't think that changes this small are even protected by copyright. :)

Ciao
Stefan

### CODE REVIEW

File: R/html.R [hunk 1]
```diff
@@ -3,7 +3,8 @@
 #' @param tufte_features A character vector of the CSS features to enable:
```
Comment (yihui) [inline R/html.R]: `margin_references` is not a CSS feature, so you need to reword this sentence.

File: R/html.R [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #' @details \code{tufte_html()} provides the HTML format based on the Tufte CSS:
 #'   \url{https://edwardtufte.github.io/tufte-css/}.
-#' @param tufte_features A character vector of the CSS features to enable:
+#' @param tufte_features A character vector of the \code{tufte_html()} features to enable:
```
Comment (yihui) [inline R/html.R]: how about `A character vector of style features`?

**Comment by yihui** [comment]
Looks good now. Could you add an entry to NEWS.md? Thanks!

**Comment by mschilli87** [comment]
@stefanfritsch: Until this is fixed, here is a possible workaround:

 1. I could reproduce this with the `bookdown::tufte_html2` output.
 2. Adding `tufte_variant: envisioned`, I did not have this problem anymore.

This changes the CSS to some extent but at least you can continue developing the document with a very close style to what you want in the end and simply switch back the style once this is fixed.

**Comment by yihui** [comment]
FYI, this caused the bug #53 so I changed the API in f9c45cc.

**Pull request merged**